### PR TITLE
Fix/repo 4906 veracode static scan

### PR DESCRIPTION
**Pull request by nithinnambiar**
Integrate Veracode static code scan with Travis.  Modified the travis file to add a new task to upload and scan alfresco.war file which generates a report on security flaws.

### CODE REVIEW

File: scripts/veracode/static_analysis.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
```
Comment (DenisGabriela) [inline scripts/veracode/static_analysis.sh]: I see [this Travis job](https://travis-ci.com/github/Alfresco/acs-packaging/jobs/299049588) is green even though the scan could not be performed. Add `set -e` to exit the script when a command fails (returns a non-zero status).

File: scripts/veracode/static_analysis_init.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+echo "=========================== Starting Static Analysis Init Script ==========================="
+
```
Comment (DenisGabriela) [inline scripts/veracode/static_analysis_init.sh]: Add `set -e` to exit the script when a command fails (returns a non-zero status).

File: .travis.yml [hunk 3]
```diff
@@ -183,7 +182,14 @@ jobs:
         - ./tests/scripts/start-compose.sh ./tests/environment/docker-compose-all-amps-test.yml
         - ./tests/scripts/wait-for-alfresco-start.sh "http://localhost:8082/alfresco"
         - travis_wait 20 mvn install -f tests/tas-all-amps/pom.xml -Pall-tas-tests -Denvironment=default -DrunBugs=false
-
+    - name: "Veracode static code scan"
```
Comment (DenisGabriela) [inline .travis.yml]: Similar to WhiteSource scan, we could run this only on master, support branches and on builds != pull_requests:
`     if: fork = false AND (branch = master OR branch =~ /support\/.*/) AND type != pull_request`.

Comment (nithinnambiar) [reply .travis.yml]: The change has been made.

File: scripts/veracode/static_analysis_init.sh (hunk 2 above)
Comment (nithinnambiar) [reply scripts/veracode/static_analysis_init.sh]: The change has been made.

File: scripts/veracode/static_analysis.sh (hunk 1 above)
Comment (nithinnambiar) [reply scripts/veracode/static_analysis.sh]: The change has been made.

**Pull request merged**